api/v1beta1/verticadb_types.go

```diff
@@ -916,3 +933,16 @@ func (s *Subcluster) GetServiceName() string {
 	}
 	return s.ServiceName
 }
+
+// RequiresTransientSubcluster checks if an online image change requires a
+// transient subcluster.  A transient subcluster exists if the template is
+// filled out or the name of the temporary routing subcluster doesn't exist.
```

Comment:
I don't quite get this line.

Reply:
If subclusters were specified by name, but none of them actually exist, I was going to have it default to creating a transient subcluster from a template.

Reply:
Ok thanks